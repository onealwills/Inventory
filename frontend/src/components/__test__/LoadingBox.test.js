import { render } from "@testing-library/react";
import LoadingBox from "../LoadingBox";

test("<LoadingBox/> snapshot test", () => {
  const component = render(<LoadingBox />);
  expect(component.container).toMatchSnapshot();
});
