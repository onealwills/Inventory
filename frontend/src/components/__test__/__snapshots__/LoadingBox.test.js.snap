// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingBox/> snapshot test 1`] = `
<div>
  <div
    class="loading"
  >
    <i
      class="fa fa-spinner fa-spin"
    />
    Wait a sec...
  </div>
</div>
`;
